app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,17 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-
+        <activity
+            android:name=".BatteryActivity"
+            android:label="@string/app_name">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+                <!-- Check if the device is connected to usb or not-->
+                <action android:name="android.intent.action.ACTION_POWER_CONNECTED"/>
```

Comment:
@adrygll2 Do we need to listen for `ACTION_POWER_CONNECTED` intents? We are only interested in scenarios where the battery is getting lower and lower, right?